warehouse/integrations/redshift/redshift.go

```diff
@@ -766,6 +781,29 @@ func (rs *Redshift) loadUserTables(ctx context.Context) map[string]error {
 		}
 	}
 
+	if !rs.ShouldMerge(warehouseutils.IdentifiesTable) {
+		identifyStagingTable = warehouseutils.StagingTableName(provider, warehouseutils.IdentifiesTable, tableNameLimit)
```

Comment:
This code looks similar to what is present in `loadTable`. Can we extract out a method and use that at both the places?